dev/integration_tests/ios_host_app/flutterapp/lib/resize

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+class ResizeApp extends StatefulWidget {
+  /// Creates the [ResizeApp].
+  const ResizeApp({super.key});
+
+  @override
+  State<ResizeApp> createState() => _ResizeAppState();
+}
+
+class _ResizeAppState extends State<ResizeApp> {
+  int _listSize = 1;
+  void _addToList() {
+    setState(() {
+      _listSize ++;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      heightFactor: 1,
+      child: Directionality(
+        textDirection: TextDirection.ltr,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(8.0, 50, 8.0, 8.0),
+              child: Semantics(
+                  label: "Add to list",
+                  child: ElevatedButton(
+                onPressed: _addToList,
+                child: Text("Add to list"),
+              )),
+            ),
+            for (int i = 0; i < _listSize; i++)
+              Text(
+                "Hello from Flutter $i",
+                style: TextStyle(color: Colors.pink, fontSize: 30),
+              ),
```

Comment:
I'd consider changing this test app to avoid text completely.

Text is super super super finicky. All kinds of things can affect your app's text. For example, Flutter might use a different font depending on the platform. Or, accessibility settings might increase the font size or weight, or increase the line hight, to improve readability. All of these might affect the size of the text, which in turn might affect the size of the dynamically sized view.

Maybe a safer example app would be a "column rainbow", a column of fixed sized boxes with different colors. Tapping on the view would add another box with the next color. This would make it less likely that the view's height would be affected by external factors.